components/app-framework/src/widgets/async-task-dialog/async-task-dialog.service.js

```diff
@@ -116,65 +116,68 @@
    * @constructor
    */
   function AsyncTaskDialogController($scope, context, content, $uibModalInstance) {
-    this.context = context;
-    this.content = content;
-    this.showSpinner = false;
-    this.$uibModalInstance = $uibModalInstance;
-    this.$scope = $scope;
+
+    var vm = this;
+
+    vm.context = context;
+    vm.content = content;
+    vm.showSpinner = false;
+
     // Set form name attribute to form.myForm
-    this.$scope.form = {};
+    vm.form = {};
```

Comment:
Is this equivalent to $scope.form ? There are other places in the file where we still refer to $scope.form - should these be changed as well?